packages/react-juce/src/lib/Stylesheet.ts

```diff
@@ -0,0 +1,62 @@
+import NativeMethods from './NativeMethods';
+
+import { all as allCssProps } from "known-css-properties";
+import camelCase from "camelcase";
+
+import Colors from "./MacroProperties/Colors";
+import { macroPropertyGetters } from "./MacroProperties";
+
+// Get any css properties not beginning with a "-",
+// and build a map from any camelCase versions to
+// the hyphenated version.
+const cssPropsMap = allCssProps
+  .filter((s) => !s.startsWith("-") && s.includes("-"))
+  .reduce((acc, v) => Object.assign(acc, { [camelCase(v)]: v }), {});
+
+//TODO: Add interface types
+export default {
+    create(stylesObj) {
+        // TODO: Will be another instance of our
+        //       styles object type.
+        let nativeStyles = {};
+
+        for (const subObj in stylesObj) {
```

Comment:
@nick-thompson , I suspect this block can be tidied further once I've added types etc. but functionally this is now doing the right thing.